wsnet/proto.go

```diff
@@ -4,18 +4,33 @@ import (
 	"github.com/pion/webrtc/v3"
 )
 
-// protoMessage is used for brokering a dialer and listener.
+// DialPolicy a single network + address + port combinations that a connection
+// is permitted to use.
+type DialPolicy struct {
+	// If network is empty, it applies to all networks.
+	Network string `json:"network"`
+	// If host is empty, it applies to all hosts. "localhost" and any IP address
+	// under "127.0.0.0/8" can be used interchangeably.
+	Host string `json:"address"`
```

Comment:
Would `Address` make more sense here? I'm concerned host is a bit ambiguous as to whether you should include the port here or not.

Reply:
Host means IP or hostname only I'll make it clear in the comment